docs/server/settings.rst

```diff
@@ -284,7 +284,19 @@ Translation environment configuration settings.
 .. versionadded:: 2.7
 
 ``POOTLE_TM_SERVER``
-  Default: Set to ``http://localhost:9200/translations``
+  Default:
+
+  .. code-block:: python
+
+    POOTLE_TM_SERVER = {
```

Comment:
`POOTLE_TM_SERVER =` is not part of the default value. In other words, you should only include the dictionary.